.pre-commit-config.yaml

```diff
@@ -11,3 +11,8 @@ repos:
         language: python
         verbose: true
         stages: [commit-msg]
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.0.264
+    hooks:
+      - id: ruff
+        # args: [ --fix, --exit-non-zero-on-fix ]
```

Comment:
Please remove this commented-out line (unless it has a good reason to be there?).